react/react_and_the_backend/project_wheres_waldo_a_photo_tagging_app

```diff
@@ -0,0 +1,32 @@
+### Introduction
+
+This project will finally give you a chance to tie together everything you've learned so far. It's a project with a lot of complexity so take it one step at a time. Working with a backend is pretty straightforward, but you'll be juggling some of the front end functionality too. This is entirely within the realm of the kind of thing you might be asked to build on the job (though of course not exactly). Are you ready???
```

Comment:
The three question marks is a bit much for me, but this is just an opinion. I'd much prefer one.